static/js/components/LocalizationPlot.jsx

```diff
@@ -363,7 +380,7 @@ const GeoJSONGlobePlot = ({
           .data(galaxies.features)
           .enter()
           .append("circle")
-          .attr("fill", (d) => (visibleOnSphere(d) ? "red" : "none"))
+          .attr("fill", (d) => (visibleOnSphere(d) ? "green" : "none"))
```

Comment:
This was just to differentiate galaxies from sources or some other reason?

Reply:
yeah!